frontend/src/metabase/css/core/text.css

```diff
@@ -111,3 +111,9 @@
     line-height: 1.4em;
     white-space: pre;
 }
+
+.text-measure { max-width: 620px; }
+
+.text-dark { color: #333333; }
```

Comment:
Looks like the #333333 one was being overridden by the real one anyway, I'm going to remove this.